### PR TITLE
Rename Databerry to Chaindesk

### PR DESCRIPTION
**Pull request by gmpetrov**
Fixes # (issue)

### CODE REVIEW

**Comment by jacoblee93** [comment]
Not sure how many folks are currently in prod with this but would it make sense to keep `DataberryRetriever` as a deprecated alias?

**Comment by gmpetrov** [comment]
> Not sure how many folks are currently in prod with this but would it make sense to keep `DataberryRetriever` as a deprecated alias?

Safe idea @jacoblee93! Will add back the DataberryRetriever class . Is there a way to mark the class as deprecated?

File: langchain/scripts/create-entrypoints.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ const entrypoints = {
   "retrievers/supabase": "retrievers/supabase",
   "retrievers/zep": "retrievers/zep",
   "retrievers/metal": "retrievers/metal",
-  "retrievers/databerry": "retrievers/databerry",
```
Comment (jacoblee93) [inline langchain/scripts/create-entrypoints.js:141]: You'll need to keep the entrypoint in for `databerry` if you want back-compat (up to you!)

**Comment by jacoblee93** [comment]
> > Not sure how many folks are currently in prod with this but would it make sense to keep `DataberryRetriever` as a deprecated alias?
> 
> Safe idea @jacoblee93! Will add back the DataberryRetriever class . Is there a way to mark the class as deprecated?

Just don't forget to put the entrypoint back in `create-entrypoints.js` as well then!

Have been doing stuff like this: 

https://github.com/hwchase17/langchainjs/blob/main/langchain/src/vectorstores/mongo.ts#L21

**Pull request closed without merging**